src/cpp/column.cpp

```diff
@@ -911,6 +911,13 @@ t_column::unintern_c(t_uindex idx) const {
     return m_vocab->unintern_c(idx);
 }
 
+std::string
+t_column::str() const {
```

Comment:
I added `str()` on columns for debugging, but we can take it out before merging. Not sure what use it would have in production.

Reply:
Ok - every byte counts!